setup.sh

```diff
@@ -16,6 +16,20 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+# Check the repo URL and main branch
+REPO_URL=$(git remote get-url origin)
+if [[ $REPO_URL == *data-team-uhn/cards-project-template.git ]]
+then
+  whiptail --backtitle "New CARDS repository setup" --title "Template repository location" --msgbox "This repository is supposed to be used as a template when creating a new repository through the github UI. You should create a new repository at github.com and select cards-project-template as the template to copy, clone the new repository, and run the setup script in there. Exiting." 10 78
+  exit 1
+fi
+# github uses two different formats for the repo url, ssh or http, while Maven expects a different format.
+# Convert the git remote URL to the format supported by Maven
+REPO_NAME=$(echo -n $REPO_URL | sed -E -e 's/.*data-team-uhn[/:](.*).git/\1/')
```

Comment:
Similar to my above comment, this line should be amended to handle the case where `$REPO_URL` does not contain the trailing `.git` or the `data-team-uhn` organization.